ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -76,6 +76,7 @@ private[sandbox] object SqlLedger {
       eventsPageSize: Int,
       metrics: Metrics,
       lfValueTranslationCache: LfValueTranslation.Cache,
+      validatePartyAllocation: Boolean = false,
```

Comment:
Likewise, does this need a default?

If it's just for tests, I'd rather the tests were updated to be explicit.

Reply:
It's not just for tests but I have made the value explicit in the test functions.